src/components/Divider/divider.module.css

```diff
@@ -0,0 +1,17 @@
+:root {
+  --denhaag-divider-border-color: var(--denhaag-grey-2);
+  --denhaag-divider-border-width: 1px;
+  --denhaag-divider-margin: 16px 16px;
```

Comment:
While in The Hague designs they margins just so happen to have the same values, this is likely not the case for other designs. I would propose to have the `margin-inline` and `margin-block` separation already in place for that.